client/sparcsssov2-node.ts

```diff
@@ -76,7 +76,7 @@ export default class Client {
     return { sign, timestamp };
   }
 
-  private _validateSign(payload, timestamp, sign) {
+  private _validateSign(payload: (string | number)[], timestamp: number, sign: Buffer) {
     const clientSign = this._signPayload(payload, false);
     if (Math.abs(clientSign.timestamp - Number(timestamp)) > 10) {
```

Comment:
`timestamp`의 type이 `number`인데, `Number` instance를 생성하는 까닭이 궁금합니다.
```suggestion
  private _validateSign(payload: (string | number)[], timestamp: number, sign: Buffer) {
    const clientSign = this._signPayload(payload, false);
    if (Math.abs(clientSign.timestamp - timestamp) > 10) {
```

Reply:
그러게요 과거의 나 왜그랬을까요